Algorithms/mentoring/.idea/workspace.xml

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please Juan, do not control ItelliJ settings files.

Reply:
This change wasn't made by my authority